### PR TITLE
Unify error types for network modifications

### PR DESCRIPTION
**Pull request by SlimaneAmar**
Signed-off-by: Slimane AMAR <slimane.amar@rte-france.com>

### CODE REVIEW

**Review by niconoir — APPROVED** [review]

File: src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java [hunk 1]
```diff
@@ -34,6 +33,7 @@ public enum ModificationType {
     DELETE_VOLTAGE_LEVEL_ON_LINE,
     DELETE_ATTACHING_LINE;
 
+    // TODO transfer method to the enum
```
Comment (archendel) [inline src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java:36]: is it planned for another PR, or is it an oversight ?

File: src/test/java/org/gridsuite/study/server/NetworkModificationTest.java [hunk 2]
```diff
@@ -984,11 +983,22 @@ public void deleteModificationRequest() throws Exception {
                         .header(USER_ID_HEADER, userId))
                 .andExpect(status().isOk());
         verifyDeleteRequest(stubId, "/v1/network-modifications", false, Map.of("uuids", WireMock.equalTo(modificationUuid.toString())));
-
         checkEquipmentDeletingMessagesReceived(studyUuid, modificationNode.getId());
         checkUpdateNodesMessageReceived(studyUuid, List.of(modificationNode.getId()));
         checkUpdateModelsStatusMessagesReceived(studyUuid, modificationNode.getId());
         checkEquipmentUpdatingFinishedMessagesReceived(studyUuid, modificationNode.getId());
+
+        String errorMessage = "Internal Server Error";
```
Comment (archendel) [inline src/test/java/org/gridsuite/study/server/NetworkModificationTest.java:991]: change message to "bad request" to avoid confusion with actual error ?

**Review by archendel — APPROVED** [review]
small remarks

File: src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java (hunk 1 above)
Comment (SlimaneAmar) [reply src/main/java/org/gridsuite/study/server/dto/modification/ModificationType.java:36]: Yes  for another PR

**Pull request merged**